AK/Forward.h

```diff
@@ -125,8 +125,12 @@ class WeakPtr;
 template<typename T, size_t inline_capacity = 0>
 requires(!IsRvalueReference<T>) class Vector;
 
+namespace Detail {
+template<typename T>
+inline constexpr bool IsAnyString = IsSame<String, RemoveCVReference<T>> || IsSame<StringView, RemoveCVReference<T>> || IsSame<char*, Decay<T>> || IsSame<char const*, Decay<T>>;
```

Comment:
Hmm it seems a bit odd to have this in a forwarding header. Since `AnyString` is the only thing using this, it could be rolled into the concept, they don't need to be separate:
```c++
template<typename T>
concept AnyString = SameAs<String, RemoveCVReference<T>> || SameAs<StringView, RemoveCVReference<T>> || SameAs<char*, Decay<T>> || SameAs<char const*, Decay<T>>;
```